### ISSUE TITLE
How to evaluate model accuracy, precision, recall in onnx format

### ISSUE THREAD

**Issue by harshakhmk** [onnx/onnx#6730]
# Ask a Question
How to evaluate model accuracy precision recall in onnx format, there are couple tutorials available for evaluating in Pytorch and tensorflow models
### Question

Is there any libarary or framework to evaluate model performance 
### Further information
- Relevant Area: 

- Is this issue related to a specific model?
**Model name**: 
**Model opset**:

**Comment by justinchuby** [comment]
You can use onnxruntime to runtime he model and provide your inputs. Everything should be standard from there. I don’t think there’s a framework implemented specially for evaluation

**Comment by harshakhmk** [comment]
I am beginner in this, Could you provide a tutorial or sample code on how to do so for sentence transformers model : all-MiniLM-L6-v2
